local.properties

```diff
@@ -0,0 +1,10 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Sun Mar 07 14:37:17 CET 2021
+sdk.dir=C\:\\Users\\mathi\\AppData\\Local\\Android\\Sdk
+#MAPS_API_KEY=AIzaSyAL2oTIMqy1leoIi791kAwjppazruq7nO8
+MAPS_API_KEY=AIzaSyAL2oTIMqy1leoIi791kAwjppazruq7nO9
```

Comment:
Is it ok for this key to be on GitHub (or is it a "secret" key) ?

Reply:
Is this the same local properties you added to git ignore? To ensure the git ignore takes effect, delete local.properties and re-create it to see if it's not commited again.